client/rest-high-level/src/test/java/org/elasticsearch/client/IngestRequestConvertersTests.java

```diff
@@ -82,29 +82,14 @@ public void testDeletePipeline() {
     public void testSimulatePipeline() throws IOException {
         String pipelineId = ESTestCase.randomBoolean() ? "some_pipeline_id" : null;
         boolean verbose = ESTestCase.randomBoolean();
-        String json = "{"
-            + "  \"pipeline\": {"
-            + "    \"description\": \"_description\","
-            + "    \"processors\": ["
-            + "      {"
-            + "        \"set\": {"
-            + "          \"field\": \"field2\","
-            + "          \"value\": \"_value\""
-            + "        }"
-            + "      }"
-            + "    ]"
-            + "  },"
-            + "  \"docs\": ["
-            + "    {"
-            + "      \"_index\": \"index\","
-            + "      \"_type\": \"_doc\","
-            + "      \"_id\": \"id\","
-            + "      \"_source\": {"
-            + "        \"foo\": \"rab\""
-            + "      }"
-            + "    }"
-            + "  ]"
-            + "}";
+        String json = """
+            {
+              "pipeline": {
+                "description": "_description",
+                "processors": [ { "set": { "field": "field2", "value": "_value" } } ]
+              },
+              "docs": [ { "_index": "index", "_type": "_doc", "_id": "id", "_source": { "foo": "rab" } } ]
```

Comment:
It looks like the original JSON was intended to be spread over more lines, except it didn't include the newlines. Maybe can put each key-pair on its own line, line the original?